lnrpc/verrpc/verrpc.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package verrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/verrpc";
+
+message VersionRequest {
+};
+
+message Version {
+    /// A verbose description of the daemon's commit.
+    string commit = 1;
+
+    /// The SHA1 commit hash of the daemon.
+    string commit_hash = 2;
+
+    /// The semantic version.
+    string version = 3;
+
+    /// The major application version.
+    uint32 app_major = 4;
+
+    /// The minor application version.
+    uint32 app_minor = 5;
+
+    /// The application patch number.
+    uint32 app_patch = 6;
+
+    /// The application pre-release modifier, possibly empty.
+    string app_pre_release = 7;
+};
```

Comment:
Build tags... ? Can be important to find out what functionality is available.

Reply:
done